src/main.ts

```diff
@@ -31,10 +31,24 @@ const COMMANDS: Record<string, Command> = {
 
 const ALLOWED_ACTIONS = ['created'];
 
+const URL_REGEX = /[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)?/gi;
+const URL_FILE_REGEX = /\.(png|jpg|jpeg|gif)$/i
+const EXCLUSION_LIST = [
+  'tachiyomi.org',
+  'github.com',
+  'user-images.githubusercontent.com',
+  'gist.github.com'
+];
+
 async function run() {
   try {
     const { eventName, repo } = github.context;
 
+    if (eventName === 'issues') {
+      await checkForDuplicates();
+      return;
+    }
+
     if (eventName !== 'issue_comment') {
```

Comment:
Can you refactor the later contents into another function so this one is also an equal check like for `issues` above?